src/components/Registration/Registration.config.js

```diff
@@ -0,0 +1,18 @@
+export const RegistrationCards = [
+  {
+    icon: "group",
+    iconLink:
+      "https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0",
+    name: "Ēnotājs",
+    text: "Lorem ipsum dolor sit amet consectetur adipisicing elit.",
+    link: "/registration/shadow",
+  },
+  {
+    icon: "work",
+    iconLink:
+      "https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0",
```

Comment:
U are importing fonts, or what does it stands for ?

Reply:
Importing Google fonts for symbols. As I mentioned in previous comment, I am not sure how to work with icons.